src/components/ui/Icon/Icon.tsx

```
@@ -1,33 +1,84 @@
+      stroke={color || "currentColor"}
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      className={`icon-svg ${className}`}
       aria-label={ariaLabel}
-      aria-hidden={!ariaLabel}
-      {...(ariaLabel && { role: 'img' })}
-    />
+      aria-hidden={ariaLabel ? "false" : "true"}
+      {...(ariaLabel && { role: "img" })}
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)